crates/server/src/datastore/postgres/migrations/2023-04-11-013700_initial_db/up.sql

```diff
@@ -33,7 +33,7 @@ CREATE TABLE records (
   id SERIAL PRIMARY KEY,
   log_id INTEGER NOT NULL REFERENCES logs(id),
   record_id TEXT NOT NULL UNIQUE,
-  checkpoint_id INTEGER REFERENCES checkpoints(id),
+  registry_log_index BIGINT UNIQUE,
```

Comment:
This could have been a separate migration, but it would have been a significant effort to actually migrate the data. Since no one yet has production data it seems reasonable to rebuild databases for now.

Reply:
Pinging @calvinrp here just so he's aware that we'll need to reset the database if this change is deployed to his dogfood server.